### PR TITLE
Update microsoft-365-mailbox-utilization-service-alerts.md

### PR DESCRIPTION
**Pull request by holliep1**
Adding a blurb that the mailbox usage report is 24 hours behind the alert. And updating a piece of the document that didn't read right.

### CODE REVIEW

File: microsoft-365/enterprise/microsoft-365-mailbox-utilization-service-alerts.md [hunk 1]
```diff
@@ -29,12 +29,14 @@ These service advisories are displayed in the Microsoft 365 admin center. To vie
 
 :::image type="content" alt-text="Mailbox utilization service alert." source="../media/MailboxUtilizationServiceAlert.png" lightbox="../media/MailboxUtilizationServiceAlert.png":::
 
-To display a list of mailboxes that are nearing their storage quota (called the *mailbox usage report*), click the highlighted link in the following screenshot. This link is displayed in the service advisory.
+To display a list of mailboxes that are nearing their storage quota, click the highlighted link in the following screenshot to access your mailbox usage report. This link is displayed in the service advisory.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/microsoft-365-mailbox-utilization-service-alerts.md]:
```suggestion
To display a list of mailboxes that are nearing their storage quota, select the highlighted link in the following screenshot to access your mailbox usage report. This link is displayed in the service advisory.
```

File: microsoft-365/enterprise/microsoft-365-mailbox-utilization-service-alerts.md [hunk 2]
```diff
@@ -29,12 +29,14 @@ These service advisories are displayed in the Microsoft 365 admin center. To vie
...
 
 :::image type="content" alt-text="Link to mailbox usage report." source="../media/LinkToMailboxUsageReport.png" lightbox="../media/LinkToMailboxUsageReport.png":::
 
 Alternatively, the direct URL to the mailbox usage report is <https://admin.microsoft.com/Adminportal/Home?source=applauncher#/reportsUsage/MailboxUsage>.
 
+Please note that the mailbox usage report information could be 24 hours behind your mailbox utilization service advisory alert.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/microsoft-365-mailbox-utilization-service-alerts.md]:
```suggestion
> [!NOTE]
> The mailbox usage report information could be 24 hours behind your mailbox utilization service advisory alert.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @holliep1, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kelleyvice-msft Please merge this PR. Thanks

**Pull request merged**